public/app/features/plugins/admin/components/Card.tsx

```diff
@@ -5,30 +5,19 @@ import { useStyles2 } from '@grafana/ui';
 
 interface Props {
   href: string;
-  text: React.ReactNode;
-  image: React.ReactNode;
-  layout?: 'horizontal' | 'vertical';
+  header: React.ReactNode;
+  content: React.ReactNode;
+  footer: React.ReactNode;
 }
 
-export const Card = ({ href, text, image, layout = 'vertical' }: Props) => {
```

Comment:
I guess the current Card component we have in grafana-ui does not work for this use-case? It seems like the one we have in grafana-ui is more like a `ListItem` component, and this one is an actual Card. I think it'd be nice if this component was moved to grafana-ui and made more generic, as we also have a similar use-case in Enterprise.

Reply:
Great feedback @Clarity-89 and I totally agree! However, is this something we can do in a follow up PR? Or do you need this component soon?

Reply:
Yep sounds like a great idea @Clarity-89. Unless I'm missing something about the current Card component we end up with some strange text layout when they're in rows.

<img width="1399" alt="Screenshot 2021-07-14 at 10 25 43" src="https://user-images.githubusercontent.com/73201/125589652-e6066efe-bd6f-4937-a503-a0eba804534d.png">

I'm guessing some configuration of spacing and / or background colour would be useful for this? Also need to think of a name. @dprokop suggested `Tile` which I quite like.

Reply:
Yeah, unfortunately card has quite specific layout, but there's also `CardContainer` component that maybe could be used here as a wrapper? https://github.com/grafana/grafana/blob/main/packages/grafana-ui/src/components/Card/CardContainer.tsx It handles the case when the card can be an `a` or a `div` and allows some custom styling, but also will keep the general styles consistent across cards.